src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/WebResponseHelper.CoreClr.cs

```diff
@@ -36,11 +36,31 @@ internal static class WebResponseHelper
         }
 
         internal static string GetOutFilePath(HttpResponseMessage response, string _qualifiedOutFile)
-        {
-            // Get file name from last segment of Uri
-            string lastUriSegment = System.Net.WebUtility.UrlDecode(response.RequestMessage.RequestUri.Segments[^1]);
+        {            
+            if (!Directory.Exists(_qualifiedOutFile))
```

Comment:
Eh, the parameter shouldn't have `_` prefix.

Reply:
I know, but I followed the convention used in `WebRequestPSCmdlet.Common.cs` when I cached QualifiedOutFile in _qualifiedOutFile

Reply:
It is not field here, it is local parameter.